### PR TITLE
Fix test for SINGULARITYENV_PATH

### PR DESCRIPTION
**Pull request by jmstover**
**Description of the Pull Request (PR):**
Fix test for SINGULARITYENV_PATH. Current test looks only to see if variable is unset, and not an empty string.

**This fixes or addresses the following GitHub issues:**

- Ref: #

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

**Review by dtrudg — APPROVED** [review]
This looks good to me.

File: etc/init [hunk 1]
```diff
@@ -11,7 +11,7 @@ unset ml
 unset BASH_ENV
 
 # Provide a sane path within the container
-if [ -z ${SINGULARITYENV_PATH+x} ]; then
+if [ -z "${SINGULARITYENV_PATH:-}" ]; then
     SINGULARITYENV_PATH="/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
 else
     SINGULARITYENV_PATH="$SINGULARITYENV_PATH:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
```
Comment (GodloveD) [inline etc/init]: I think this whole `if` statement can be thrown away and replaced by a single line.  
```
SINGULARITYENV_PATH="{$SINGULARITYENV_PATH:-}:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
```
That should do the right thing, correct?

**Review by GodloveD — CHANGES_REQUESTED** [review]
I think this change can be simplified.  Plz correct me if I am wrong.

File: etc/init (hunk 1 above)
Comment (dtrudg) [reply etc/init]: With that formulation you'd have a lone : at start of it if the var isn't set when the test occurs.

Also, $ is outside of {

Comment (GodloveD) [reply etc/init]: Ah. Right you are. (On both counts of course).

**Review by GodloveD — APPROVED** [review]

**Pull request merged**